.gitignore

```diff
@@ -7,4 +7,7 @@ target/
 
 metastore_db/
 scylla
-derby.log
\ No newline at end of file
+derby.log
+
+/Makefile.main
+/.ci
```

Comment:
Unexpected EOF

Reply:
how can I reproduce it? travis works just fine